.github/workflows/e2e.yaml

```diff
@@ -93,35 +93,12 @@ jobs:
         # output_dir
         mkdir -p _artifacts
         # create cluster
-        cat <<EOF | /usr/local/bin/kind create cluster \
+        sed -i  's/IP_FAMILY/'${IP_FAMILY}'/g' examples/kind/kind.config
+        sed -i  's#EGREE_SELECTOR_CONFIGURATION_PATH#./examples/kind/egress_selector_configuration.yaml#g' examples/kind/kind.config
```

Comment:
does this work? kind.config does not have those values, does it?

Reply:
Thanks,Apparently I forgot to submit this file.